scout/server/blueprints/variants/templates/variants/fusion-variants.html

```diff
@@ -123,7 +123,7 @@
     <td><span data-bs-toggle="tooltip" data-bs-html="true" title="{% for name, caller in variant.callers %}{{ name }}: {{ caller }}<br>{% endfor %}">
       {{ variant.tool_hits|int }}</span></td>
     <td class="text-end">{{ variant.fusion_score|float|round(3) }}</td>
-    <td>{{ variant.samples[0].read_depth }}</td>
+    <td>{{ variant.samples[0].read_depth if variant.samples[0].read_depth != -1 else 0 }}</td>
```

Comment:
Why not changing the parsing function so it saves 0 instead of -1?

Reply:
I couldn't understand the logic of that part of the code and I was afraid of affecting the parsing of other VCF files. I just thought that this was easier

Reply:
Ah OK, but read_depth = -1 is kind of weird.. Let's investigate the parsing

Reply:
This function: https://github.com/Clinical-Genomics/scout/blob/a95fb214586f87f31460273beb58324b1f500297/scout/parse/variant/genotype.py#L292

Reply:
it's only -1 when 0 or .

Reply:
Checking the demo instance fusion variantS page you have the last variant with -1:

<img width="719" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/b1ca803c-058e-410c-a2a4-56b5253230b5">

But if you check the VCF the same variant looks like this:

3	18858170	.	N	N]3:22523820]	.	PASS	SVTYPE=BND;CHRA=0;CHRB=0;GENEA=MED30;GENEB=SLC30A8;POSA=0;POSB=0;ORIENTATION=nan,nan;FOUND_DB=;FOUND_IN=arriba;TOOL_HITS=1.0;SCORE=0.062;FRAME_STATUS=nan;TRANSCRIPT_ID_A=nan;TRANSCRIPT_ID_B=nan;TRANSCRIPT_VERSION_A=0;TRANSCRIPT_VERSION_B=0;HGNC_ID_A=23032;HGNC_ID_B=0;EXON_NUMBER_A=0;EXON_NUMBER_B=0;ANNOTATIONS=nan	`GT:DV:RV:FFPM	./1:0:0:0.0`

Reply:
Ok this is confusing, When I load the variants in the demo, and print what is actually parsed by the [parse_genotype function](https://github.com/Clinical-Genomics/scout/blob/a95fb214586f87f31460273beb58324b1f500297/scout/parse/variant/genotype.py#L45) get this:

{'individual_id': '888-88', 'display_name': '888-88', 'genotype_call': './1', 'so': None, 'alt_depth': 4, 'ref_depth': -1, 'read_depth': 4, 'alt_frequency': -1.0, 'genotype_quality': -1, 'ffpm': 8, 'split_read': 0}

{'individual_id': '888-88', 'display_name': '888-88', 'genotype_call': './1', 'so': None, 'alt_depth': 1, 'ref_depth': -1, 'read_depth': 1, 'alt_frequency': -1.0, 'genotype_quality': -1, 'ffpm': 2613, 'split_read': 0}

{'individual_id': '888-88', 'display_name': '888-88', 'genotype_call': './1', 'so': None, 'alt_depth': 4, 'ref_depth': -1, 'read_depth': 4, 'alt_frequency': -1.0, 'genotype_quality': -1, 'ffpm': 1, 'split_read': 0}

{'individual_id': '888-88', 'display_name': '888-88', 'genotype_call': './1', 'so': None, 'alt_depth': 2, 'ref_depth': -1, 'read_depth': 2, 'alt_frequency': -1.0, 'genotype_quality': -1, 'ffpm': 0, 'split_read': 0}

{'individual_id': '888-88', 'display_name': '888-88', 'genotype_call': './1', 'so': None, 'alt_depth': -1, 'ref_depth': -1, 'read_depth': -1, 'alt_frequency': -1.0, 'genotype_quality': -1, 'ffpm': 0, 'split_read': 0}

The ref depth is always -1 and the genotype of the variants VCF is like this:

```
GT:DV:RV:FFPM	./1:4:0:8.0339
GT:DV:RV:FFPM	./1:1:0:2613
GT:DV:RV:FFPM	./1:4:0:1
GT:DV:RV:FFPM	./1:2:0:0.0159
GT:DV:RV:FFPM	./1:0:0:0.0
```

Reply:
I'm just trying to understand since I'm not familiar with fusion data..

Reply:
The fact that alt_depth of the last variant is -1 despite the 0s on the VCF could also be a bug in scout!

Reply:
@fevac I've opened this PR to fix the parsing: https://github.com/Clinical-Genomics/scout/pull/4282/files